net-p2p/dogecoin/dogecoin-1.14.4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Dogecoin (unstable) 1.14.4 Development version with new code for future release."
```

Comment:
To add to https://bugs.gentoo.org/811492 comments :
> Please be more specific in the DESCRIPTION. The objective description may be <=80 characters long.

To be more explicit about the description, I can suggest  : *"Dogecoin Core ${PV} development version of blockchain node and RPC server."*

You can inspire yourself from the introduction sentence of the repository, it's going to be rewrite : https://github.com/dogecoin/dogecoin/pull/2570